### ISSUE TITLE
[SYNTH] Rumble: a chord & swarm synth

### ISSUE THREAD

**Issue by hox3d** [topisani/OTTO#109]
Since a nice picture is always better:

![rumble](https://user-images.githubusercontent.com/25805149/53867487-3e38f100-3ff4-11e9-90ed-c2e55ed70996.png)

So. That's basically a **three oscillators machine with variable waveshape and built-in drift**.
The idea behind it is to create wavy pads, dub chords, etc. It can also be used as a swarm synth: if the three oscillators are at the same pitch, slightly detuning them should create interesting evolving textures.

The noise source (Perlin or whatever) is going straight into a delay line (sample the noise three times every n samples), so that the oscillators don't have the same detune amount at the same moment.
The noise control is a macro-control: it changes both noise amplitude and speed. The curve will be adapted empirically (good compromise between speed and amount).

The first oscillators tracks the keyboard, while the other do the same but with an offset. This offset should be continuously variable if possible (i.e going in semitone steps when turning the encoder fast, and fine tune when turning the encoder slower).

So let me know what you think!

Oh, and about that UI: all ideas welcome, that was just a quick draft. But for the record: flux shows the noise generation (the road), the gauge shows the MPH (pitch of oscillators). and there is something to be found for shape (maybe a piston following the shape).

I'll try to write the code soon.

**Comment by topisani** [comment]
@jmidt I think this is could be a very nice design of a synth that's purpose-driven instead of implementation driven. 
I'd love to see an implementation, this could be a nice place for someone to start writing engines!

**Comment by hox3d** [comment]
Yeah yeah I know, I tend to start many things and never finish them.
I should have time soon to dive into OTTO code.

How about that "blank engine" template?
